Assets/Scripts/Player/PlayerControls.cs

```diff
@@ -243,7 +245,12 @@ public override void Update()
 			     player.stats.CanUseStat( Stat.Stamina ) &&
 			     player.physics.ClimbCheck() )
 			{
-				player.physics.ClimbSurface( player.controls.GetMoveInput() );
+				Vector3 moveInput = player.controls.GetMoveInput();
+				player.physics.ClimbSurface( moveInput );
+				player.animator.SetFloat( "moveSpeed", moveInput.magnitude );
+				player.animator.SetFloat( "verticalSpeed", moveInput.z );
+				player.animator.SetFloat( "horizontalSpeed", moveInput.x );
+				player.animator.SetFloat( "slope", Mathf.Abs( Vector3.Dot( moveInput, Vector3.forward ) ) );

```

Comment:
Also watch out because you might want transform.forward instead of vector3.forward.